### PR TITLE
Change ID type from int to str

### PR DESCRIPTION
**Pull request by YevheniiSemendiak**

### CODE REVIEW

**Comment by YevheniiSemendiak** [comment]
ping @artemlops 🙂

File: cocorepr/cli.py [hunk 1]
```diff
@@ -128,15 +128,15 @@ def main(args=None):
         logger.info('Using coco_crop dataset only.')
         coco = coco_crop
 
+    if drop_invalid_elements:
+        coco = remove_invalid_elements(coco)
+        logger.info(f'After removing invalid elements: {coco.to_full_str()}')
+
```
Comment (atemate) [inline cocorepr/cli.py:134]: side change but I think it should not affect the logic. tests green.

**Review by atemate — APPROVED** [review]
Thanks!

File: cocorepr/cli.py (hunk 1 above)
Comment (YevheniiSemendiak) [reply cocorepr/cli.py:134]: it actually does, if there are some invalid elements and `max_crops_per_class` is used - cocorepr crashes.
however, if we firstly filter out invalids, and then cut off redundant crops - all is good

**Pull request merged**